src/executables/utility.h

```diff
@@ -738,6 +739,18 @@ typedef enum
   FLASHBACK_MSG_USAGE = 60
 } MSGCAT_FLASHBACK_MSG;
 
+/* Message id in the set MSGCAT_UTIL_SET_MEMMON */
+typedef enum
+{
+  MEMMON_MSG_EXCLUSIVE_OPTION = 1,
+  MEMMON_MSG_NOT_SUPPORTED = 2,
+  MEMMON_MSG_NO_MATCHING_MODULE = 3,
+  MEMMON_MSG_INVALID_TRAN_COUNT_OPTION = 4,
+  MEMMON_MSG_NEGATIVE_TRAN_COUNT_NUMBER = 5,
```

Comment:
```suggestion
  MEMMON_MSG_INVALID_TRAN_COUNT_NUMBER = 5,
```

Is it enough to check only negative value?

Reply:
Of course. '-c' option just needs positive integer value, and cubrid filters overflow case before it occurs error.

Reply:
Please let me know what is the result in the following cases.
- --tran-count INT_MAX
- --tran-count INT_MAX+1

Reply:
--tran-count INT_MAX -> It works
--tran-count INT_MAX+1 -> negative error

Reply:
Done.